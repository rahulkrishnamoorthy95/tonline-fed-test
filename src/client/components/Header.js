import React from 'react';

function Header() {
  return (
    <header id="header">
        <div classname="card-header">
            <h1>{'Cardzzz!'}</h1>
        </div>
    </header>
  );
}

export default Header;
