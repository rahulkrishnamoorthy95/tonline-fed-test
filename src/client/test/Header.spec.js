import React from 'react';
import Header from '../components/Header';

describe('Header', () => {
	it('renders Cardzzz!', () => {
		const actual = shallow(<Header />);
		expect(actual.get(0)).toMatchSnapshot();
	});
});
