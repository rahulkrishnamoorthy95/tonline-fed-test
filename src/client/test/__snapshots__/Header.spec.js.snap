// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders Cardzzz! 1`] = `
<header
  id="header"
>
  <div
    classname="card-header"
  >
    <h1>
      Cardzzz!
    </h1>
  </div>
</header>
`;
